### ISSUE TITLE
Text does not maintain placement

### ISSUE THREAD

**Issue by Pozzyx2019** [Tercioo/Details-Damage-Meter#119]
Hi, in the title bar, the one that rests Dps or Damage Done, in the y options of the text I center it, but when I re-log another Pg or the same the text goes to the initial position. That is, it does not maintain centering.

https://ibb.co/YBd4k5k 
text 3 no good

https://ibb.co/KXK6k50
Ok 1 on y text ok

and when relog , the text return valor 3.....any idea how can fix this?

**Comment by UltraVolta425** [comment]
I would like to confirm this, I have the exact same issue:

https://imgur.com/a/1hI3MMX

In the above screenshot it's okay, but after a /RL or a relog it just goes back to some other values:

https://imgur.com/a/8YNrInW

**Comment by Pozzyx2019** [comment]
yes me too, same problem....need fix i hope next version...by

**Comment by Tercioo** [comment]
Using version 7927 
<https://www.curseforge.com/wow/addons/details/files/3084546> from 
twitch app?

------ Original Message ------

**Comment by Pozzyx2019** [comment]
yes last version....

**Comment by Pozzyx2019** [comment]
wait no alpha version...but last on site curseforge

**Comment by Pozzyx2019** [comment]
9.0.1.7927.143
 Scarica

**Comment by UltraVolta425** [comment]
I'm using the following versin: DetailsRetail.9.0.1.7927.143

**Comment by fastlane27** [comment]
I can confirm this issue as well on 9.0.1.7927.143

**Comment by UltraVolta425** [comment]
The issue seems to persist through version 9.0.1.7930.143.

**Comment by Pozzyx2019** [comment]
yes , same error text reset when moving in y position......not fix

**Comment by tarontaz** [comment]
Same as the above.
After I put my title bar text in a central position ; this only lasts until a reload or relog at which point it resets to default position.

v.9.0.1.7908.143

**Comment by tarontaz** [comment]
**_This remains an issue both in the current retail release (v.9.0.1.7930)  and in the Shadowlands release (v.9.0.1.7908.143)_**


**Properly placed text before reload :**
![image](https://user-images.githubusercontent.com/52515676/97000672-fa7b5e00-152e-11eb-94bf-34db64104222.png)


**Resets position after reload :**
![image](https://user-images.githubusercontent.com/52515676/97000712-08c97a00-152f-11eb-81a2-e8ec7efe5fbe.png)

**Comment by tarontaz** [comment]
**Note: Problem persists for v.9.0.1.7932.143**

**Comment by tarontaz** [comment]
**Note: Problem persists for v.9.0.1.7933.143**

**Comment by Tercioo** [comment]
I tested this with a clean install and the issue does not happen.
I'm running out of ideas of what could be causing the issue.

------ Original Message ------

**Comment by Pozzyx2019** [comment]
sorry what do you mean, clean install?

I install it, manually if you want, download the file from here and unzip it, in the addon folder ...but i don't think it solves the problem, and anyway, it's not the only thing that doesn't work,, custom text doesn't work, 
later I will post the images, with the problem ... thanks anyway for the answer ,,,

**Comment by tarontaz** [comment]
To test the notion of a fresh install; I removed all trace of any Details files (including plugins) 
in both my Retail (and Beta folders just in case). I then installed fresh **Details-DetailsRetail.9.0.1.7933.143**
from WowUp.

**The problem persists with a fresh install :**
![image](https://user-images.githubusercontent.com/52515676/97243568-8242c000-17ee-11eb-928a-302197ee1363.png)

**After /reload ui :**
![image](https://user-images.githubusercontent.com/52515676/97243665-bb7b3000-17ee-11eb-9c65-357777f71005.png)

**I have tried this on a level 1 toon with no other addons enabled - with the same result :(**

**Comment by UltraVolta425** [comment]
Thank the LORD! It actually maintains its position now after a /rl! nice one mate! :)

**Comment by Pozzyx2019** [comment]
yes good job ...tercio

**Comment by tarontaz** [comment]
**Confirm fixed; great job :)**
